coredns/tests/common.py

```diff
@@ -84,6 +84,7 @@
     NAMESPACE + '.go.info',
     NAMESPACE + '.go.memstats.gc_cpu_fraction',
     NAMESPACE + '.go.memstats.heap_released_bytes',
+    NAMESPACE + '.cache_request_count',
```

Comment:
This metric is not in v1.7, so tests are failing.
You can probably rename this variable to METRICS_V1_8 and use 1.8.5 in testing [here](https://github.com/DataDog/integrations-core/blob/master/coredns/tox.ini#L5-L19)